pkg/interface/src/logic/reducers/contact-update.ts

```diff
@@ -59,18 +59,16 @@ const edit = (json: ContactUpdate, state: S) => {
     if (!field) {
       return;
     }
-    const contact = state.contacts?.[ship];
+
     const value = data['edit-field'][field];
-    if(!contact) {
-      return; 
-    }
     
     if(field === 'add-group') {
-      contact.groups.push(value);
+      state.contacts[ship].groups.push(value);
     } else if (field === 'remove-group') {
-      contact.groups = contact.groups.filter(g => g !== value);
+      state.contacts[ship].groups =
+        state.contacts[ship].groups.filter(g => g !== value);
     } else {
-      contact[field] = value;
+      state.contacts[ship][field] = value;
```

Comment:
Why would stating it directly instead of using the assignment remedy this?

Reply:
This is a simplification, not a part of the bug fix. I did remove the unnecessary !contact check from the logic, though. The meat of this fix is in the "hasLoadedAllowed" state hook. Happy to chat through it if you want to call.

Reply:
I'm just gonna try AA's specified repro with this branch